models/us_cms_open_payments/code/check_sources.py

```
@@ -0,0 +1,63 @@
+"""Confirm every catalogued source URL resolves before any download starts."""
+
+import concurrent.futures as cf
+import subprocess
+
+import constants as c
+
+
+def status(url: str) -> str:
+    out = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)